src/snowflake/cli/_plugins/spcs/services/commands.py

```diff
@@ -297,6 +298,131 @@ def logs(
     return StreamResult(cast(Generator[CommandResult, None, None], stream))
 
 
+@app.command(requires_connection=True)
+def events(
+    name: FQN = ServiceNameArgument,
+    container_name: str = typer.Option(
```

Comment:
Quite a few of these flags are defined in exactly the same way - could you define them once outside the function and refer them here? You can check `_plugins.git.commands` for an example